source/util/bit_stream.h

```diff
@@ -17,6 +17,7 @@
 #ifndef LIBSPIRV_UTIL_BIT_STREAM_H_
 #define LIBSPIRV_UTIL_BIT_STREAM_H_
 
+#include <algorithm>
```

Comment:
For std::reverse.

It turns out we also need cassert to make this self contained.  Can be in another CL.